modules/soe_paragraph_stories/config/install/core.entity_view_display.paragraph.stanford_stories.default.yml

```diff
@@ -41,49 +40,53 @@ third_party_settings:
         - stanford_stories_node_link
       cta:
         - stanford_stories_cta_link
+      border_color_variant:
+        - stanford_stories_photo_border
```

Comment:
I moved this into it's own field as it was much cleaner/easier to parse this way.